docs/install.md

```diff
@@ -11,7 +11,7 @@ The installation of DietPi consists of few steps:
 - Get the DietPi image (and put it on the installation media)
 - Boot up the DietPi device and go through one time installation steps
 
-Following these steps you will be able to initially setup DietPi and install additional software packages you would like to use, using [`dietpi-software`](../dietpi_tools/#dietpi-software).
+Following these steps you will be able to initially setup DietPi and install additional software packages you would like to use, using [dietpi-software](../dietpi_tools/#dietpi-software){:class="nospellcheck"}.
```

Comment:
It shouldn't be required here, also as it's an actual console command, having it in code tags may even suite?

Reply:
However, not a blocker at all, just thinking for those DietPi commands in general.